source/_docs/logs.md

```diff
@@ -34,7 +34,7 @@ Log files track and record your site's activity to help you find, debug, and iso
         <td>Webserver error log.</td>
       </tr>
       <tr>
-        <th>php-error.log</th>
+        <th>php-error.log <a class="pop" rel="popover" data-proofer-ignore data-toggle="popover" data-html="true" data-content="The PHP error log is provided in each environment on the <strong>Errors</strong> tab of the Site Dashboard. For details, see <a href='/docs/php-errors'>PHP Errors and Exceptions</a>."><em class="fa fa-info-circle"></em></a></th>
         <td>1MB of log data</td>
         <td>PHP <a href="http://php.net/manual/en/book.errorfunc.php">fatal error log</a>; will not contain stack overflows. Errors from this log are also shown in the Dashboard.</td>
       </tr>
```

Comment:
How about adding more rows to this table for other types of logs and removing some of the questions below?

Drupal database log | instructions on how to access
WordPress error log | instructions to enable/access
Varnish logs | not available
Solr logs | not available